tools/tertiary-analysis/sceasy/sceasy_macros.xml

```diff
@@ -0,0 +1,24 @@
+<macros>
+  <token name="@TOOL_VERSION@">0.0.1</token>
```

Comment:
shouldn't we maybe use the tool version of the original tool here? then is easier to see in Galaxy what version of the underlying tool is being used (0.0.5) in this case.